hygen/_templates/layer/new/inject_view_useEffect.ejs.t

```diff
@@ -1,35 +1,30 @@
 ---
 inject: true
-to: "<%= attach ? `src/components/views/${view}.js` : null %>"
-before: return \(
-skip_if: "id: '<%= h.changeCase.camelCase(name) -%>'"
+to: "<%= attach ? `src/${view_path}` : null %>"
+before: "// Auto import useEffect"
+skip_if: "addSource(<%= h.changeCase.camelCase(source_file) %>)"
 ---
 
-  const SOURCE_ID = `<%= h.changeCase.camelCase(name) + 'Source' -%>`;
-  const LAYER_ID = `<%= h.changeCase.camelCase(name) -%>`;
+  const LAYER_ID = `<%= h.changeCase.camelCase(name) -%>Layer`;
```

Comment:
@albertoarana I've added this: `answers.name = answers.name.replace('Layer', '').replace('layer', '');`. I think it's easier to allow user to optionally avoid write layer or source (in source generator). However, in the new task about layer generator, I've added that `layer` word is automatically added to name to avoid to add it in .ejs.t. If you allow user to write anything, it's going to be a mess. It's better to add a standard suffix in my opinion.